### PR TITLE
KnowledgeBase Bot Rules

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Added rule policy to config, removed TED and Memoization policies
- Converted existing stories to rules.yaml
- Updated NLU data to 2.0 yaml format

### CODE REVIEW

**Comment by akelad** [comment]
ok i think in this situation it actually makes sense to just have rules! Have you checked whether this still works with the knowledge base actions in their current format? I don't think anything has changed on that front yet

File: examples/knowledgebasebot/data/nlu.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      does the [second one]{"entity": "mention", "value": "2"} [include breakfast]{"entity": "attribute", "value": "breakfast-included"}?
+      what is the [price range]{"entity": "attribute", "value": "price-range"} of the [second]{"entity": "mention", "value": "2"} hotel?
+      does the [first]{"entity": "mention", "value": "1"} one have [wifi]{"entity": "attribute", "value": "free-wifi"}?
+      does the [third]{"entity": "mention", "value": "3"} one have a [swimming pool]{"entity": "attribute", "value": "swimming-pool"}?
+      what is the [star rating]{"entity": "attribute", "value": "star-rating"} of [Berlin Wall Hostel]{"entity": "hotel"}?
+      Does the [Hilton]{"entity": "hotel"} have a [swimming pool]{"entity": "attribute", "value": "swimming-pool"}?
+
+  - lookup: restaurant
+    examples: |
+      Donath
```
Comment (akelad) [inline examples/knowledgebasebot/data/nlu.yaml:60]: same thing here as in your other PR

**Comment by akelad** [comment]
@b-quachtran make sure to tag your PR with `status:ready-to-merge` next time so it auto merges :)

**Comment by b-quachtran** [comment]
Yeah I wasn't sure about merging since one of the CI checks failed

**Pull request merged**